internal/ignition/templates/iscsi-oci-configure-secondary-nic.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+chassis_asset_tag="$(dmidecode --string chassis-asset-tag)"
+if [ "${chassis_asset_tag}" != "OracleCloud.com" ]
+then
+  echo "Not running in Oracle Cloud Infrastructure. Skipping."
+  exit 0
+fi
+
+if [ ! -d "/sys/firmware/ibft" ]
+then
+  echo "No IBFT configuration found. Skipping."
+  exit 0
+fi
+
+MTU=9000
```

Comment:
you can't hardcoded this. 

there are network with smallar / and larger packest and on iSCSI frame you have `DF` bit set. which can cause issues.

you can take it from the Static network config

Reply:
this is the configuration for the other interface, that will be connected to the load balancer.

OCI instances use a MTU of 9000 by default:
https://docs.oracle.com/en-us/iaas/Content/Network/Troubleshoot/connectionhang.htm#Overview
```
All OCI compute instances use an MTU of 9000 by default
```

If the default value (1500 I guess), will PMTUD increase the size of the MTU if needed?

> you can take it from the Static network config

what do you mean? It's not specified in the config send by the API.

Reply:
just wanted to be more generic